src/Uno.UI/UI/Xaml/Media/Brush.Android.cs

```diff
@@ -93,6 +94,22 @@ internal static IDisposable AssignAndObserveBrush(Brush b, ColorSetterHandler co
 				var disposables = new CompositeDisposable(5);
 				imageBrushCallback();
 
+				var imageBrushSource = imageBrush.ImageSource;
+				if (imageBrushSource is SvgImageSource svgImageSource)
+				{
+					svgImageSource.RegisterDisposablePropertyChangedCallback(
+						SvgImageSource.UriSourceProperty,
+						(_, _) => imageBrushCallback()
+					).DisposeWith(disposables);
```

Comment:
```suggestion
					svgImageSource
						.RegisterDisposablePropertyChangedCallback(SvgImageSource.UriSourceProperty, ImageBrushCallback)
						.DisposeWith(disposables);
```

Reply:
and define this local function:
```cs
void ImageBrushCallback(DependencyObject sender, DependencyPropertyChangedEventArgs e) => imageBrushCallback();
```
and, you can replace rest of lambda funcs too